packages/framework-provider-azure-infrastructure/src/infrastructure/helper/az-cli-helper.ts

```diff
@@ -1,14 +1,40 @@
 import { ResourceGroup } from '../types/resource-group'
 import { runCommand } from '../../../../framework-common-helpers'
+import { Resource } from '../types/resource'
 
 const azCommand = 'az'
 
 export async function getResourceGroup(appName: string): Promise<ResourceGroup> {
-  const resourceGroupName = `resource-group-${appName}-azure`
+  const environment = process.env.BOOSTER_ENV ?? 'azure'
+  const resourceGroupName = `resource-group-${appName}-${environment}`
```

Comment:
I changed this method in the Terraform PR as we need to set this name in a specific way for Terraform. Let's check that we set the correct one before merging it